v23.1/execution-locality-backups.md

```diff
@@ -0,0 +1,98 @@
+
+For an overview of the backup job phases, see the [Backup Architecture](backup-architecture.html) page.
+
+## Examples
+
+This section outlines some possible use cases for `EXECUTION LOCALITY` option.
+
+### Access backup storage restricted by network rules
+
+In certain cases, a cluster could be restricted by network rules across regions (or another locality requirement) for security purposes. This means that only nodes within the same region as the backup [cloud storage](use-cloud-storage.html) location can access the storage bucket. With `EXECUTION LOCALITY`, you can set up the backup to pin execution of the job to only the nodes with access to the bucket.
+
+The following diagram shows a CockroachDB cluster where each of the nodes can communicate with each other through a specific port, but any other network traffic between regions is blocked. Leaseholders in regions that do not match the cloud storage region (Node 2) cannot access the storage to [export backup data](backup-architecture.html#export-phase). 
+
+Instead, Node 3's locality matches a backup job created with `EXECUTION LOCALITY`. Leaseholders and replicas within that locality designation holding the row data to back up will begin reading and exporting to cloud storage.
+
+<img src="{{ 'images/v23.1/network-restriction.png' | relative_url }}" alt="Using execution locality when there is a network restriction between locality requirements" style="border:0px solid #eee;max-width:100%" />
+
+To execute the backup from only the region with access to cloud storage, you can specify [locality filters](cockroach-start.html#locality) that a node must match to take part in the backup job's execution.
+
+For example, you can pin the execution of the backup job to `us-west-1`: 
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BACKUP DATABASE {database} INTO 'external://backup_storage_uswest' WITH EXECUTION LOCALITY = 'region=us-west-1';
+~~~
+
+See the [`BACKUP`](backup.html) page for further detail on other parameters and options. 
+
+### Create a non-primary region for backup jobs
+
+When your [multi-region](multiregion-overview.html) cluster is running a frequent full backup and incremental backup schedule, it is necessary to balance the [backup schedule](create-schedule-for-backup.html) as well as the impact backup jobs may have on a cluster running data-intensive applications. One solution is to use node locality to section off part of the cluster, which will run backups, to minimize the impact running jobs have on the cluster.
```

Comment:
I'm not sure if we'd go so far as to say " it is necessary" -- maybe letting the jobs or the workload hog everything is fine for some users.

I might present this as more a "here's an option in case you want it" sort of thing, e.g. "Restricting the execution of backup jobs -- which can sometimes consume considerable resources when running full and incremental backups simultaneously -- to a designated subset of nodes that are node used to serve application traffic can be one approach to minimizing the impact the running background jobs have on the foreground application traffic." 🤷